docs/src/modules/components/ThemeContext.js

```diff
@@ -224,6 +225,15 @@ export function ThemeProvider(props) {
         spacing,
       },
       dense ? highDensity : null,
+      {
+        components: {
+          MuiCssBaseline: {
+            styleOverrides: {
+              body: paletteMode === 'dark' ? darkScrollbar() : null,
```

Comment:
😢

Reply:
Are you saying you would rather not see it used in the documentation? Most of the audience is on Windows. Maybe I'm not troubled by the behavior because I have my scrollbar configured to behave like on Windows.

<img width="366" alt="Capture d’écran 2021-02-08 à 00 54 05" src="https://user-images.githubusercontent.com/3165635/107163135-30dacd00-69a8-11eb-8cf2-facb5af510e4.png">

Reply:
Not personally, but I get it, which is why I didn't comment further.